### PR TITLE
Model validations

### PR DESCRIPTION
**Pull request by AyoMoses1**
I worked on the following
Added the following validations:
For the User model:
- Name must not be blank.
- PostsCounter must be an integer greater than or equal to zero.
For the Post model:
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.
- Added unit specs for all of your models' methods and validations.
Please feel free to review and drop your feedbacks where necessary

### CODE REVIEW

File: spec/model/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    )
+  end
+
+  it 'name must not be blank' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'postcounter must be an integer greater than or equal to zero' do
+    subject.postscounter = -1
```
Comment (tufoinnkuo10) [inline spec/model/user_spec.rb:32]: - [ ] Great job so far. Please, if you check your guidelines well, it says you should add unit specs for all of your models' methods and validations. Please kindly add test for the `Three most recent post` method in the user model

File: spec/model/post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  it 'likescounter must be an integer greater than or equal to zero' do
+    subject.likescounter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'updates user posts counter' do
+    subject.save
+    expect(user.postscounter).to eq(1)
+    Post.create(title: 'Second Anything', text: 'Second Anything test', author: user, commentscounter: 0,
+                likescounter: 0)
```
Comment (tufoinnkuo10) [inline spec/model/post_spec.rb:46]: - [ ] please kindly add test for the `five recent comments` method you have in your post model. nice work.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @AyoMoses1 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Gitflow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review

- [Required] Please your MIT link in the  Readme file is broken. kindly fix and add a working one for your file to be professional.

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by sumairq — APPROVED** [review]
### Hi @AyoMoses1 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**